pkg/rtc/transport.go

```diff
@@ -263,7 +263,7 @@ func newPeerConnection(params TransportParams, onBandwidthEstimator func(estimat
 		directionConfig.RTPHeaderExtension.Video = append(directionConfig.RTPHeaderExtension.Video, rtpextension.PlayoutDelayURI)
 	}
 
-	me, err := createMediaEngine(params.EnabledCodecs, directionConfig)
+	me, err := createMediaEngine(params.EnabledCodecs, directionConfig, params.IsOfferer)
```

Comment:
In tests, the offerer is the publisher. So, this will filter out H.264 high profile in UTs. I think it should be fine. But, maybe make an explicit argument here. We already use `IsOfferer`. I remember calling it `IsOfferer` and making that change a while back because it was a bit confusing at that point. It is still probably confusing, but I am thinking it is okay if UTs do not offer H.264 high profile.